app/services/challenge/process_expired_challenges.rb

```diff
@@ -11,7 +11,7 @@ def call
   def delete_challenge_progresses
     @expired_challenges.each do |c|
       completed_challenges = ChallengeProgress.where('challenge_id = ?', c)
-      p "Removing... #{c.stringify}"
+      Rails.logger.debug { "Removing... #{c.stringify}" }
```

Comment:
Using Rails logger allows debug output to be turned off in some environments. Passing a string inside a block, as opposed to directly passing a string, is more performant.